utils/sub8_ros_tools/sub8_ros_tools/geometry_helpers.py

```diff
@@ -70,3 +73,36 @@ def compose_transformation(R, t):
     transformation[3, :3] = t
     transformation[3, 3] = 1.0
     return transformation
+
+
+def quat_to_rotvec(q):
+    ''' Turn a quaternion into a axis rotation vector [x,y,z] '''
+    if q[3] < 0:
+        q = -q
+    q = transformations.unit_vector(q)
+    angle = math.acos(q[3])*2
+    axis = normalize(q[0:3])
```

Comment:
I should have noted this in the normalize docs.

ros_tools.normalize is useful for some graphics applications in simul8, where having a zero vector having zero direction is useful.

Forrest: Is the reason you made a random vector because we don't care about the axis if there's no rotation? Any reason not default to pooping out unit-Z?